lib/utils/matrix_sdk_extensions/download_file_extension.dart

```diff
@@ -81,7 +82,7 @@ extension DownloadFileExtension on Event {
         },
         cancelToken: cancelToken,
       );
-      if (downloadResponse.statusCode == 200) {
+      if (downloadResponse.statusCode == 200 && await File(savePath).exists()) {
```

Comment:
can we need another verify here? compare file size?

Reply:
okay, we should do that